$OpenBSD: patch-src_network_ssl_qsslsocket_cpp,v 1.4 2015/06/03 07:28:01 dcoppa Exp $
Enable picking up certs from /etc/ssl/certs.pem and its neighbors.
--- src/network/ssl/qsslsocket.cpp.orig	Thu May  7 16:14:44 2015
+++ src/network/ssl/qsslsocket.cpp	Sun May 31 13:28:29 2015
@@ -2330,7 +2330,8 @@ QList<QByteArray> QSslSocketPrivate::unixRootCertDirec
                                << "/var/ssl/certs/" // AIX
                                << "/usr/local/ssl/certs/" // Solaris
                                << "/etc/openssl/certs/" // BlackBerry
-                               << "/opt/openssl/certs/"; // HP-UX
+                               << "/opt/openssl/certs/" // HP-UX
+                               << "/etc/ssl/"; // OpenBSD
 }
 
 QT_END_NAMESPACE
